api/types/constants.go

```diff
@@ -696,3 +700,8 @@ const (
 	// already existing users are not deleted
 	TeleportServiceGroup = "teleport-system"
 )
+
+const (
+	// This is the `Description`` of a SAML IdP Service Provider to show when listing it in the WebUI.
+	SAMLIdPServiceProviderDescription = "SAML Application"
```

Comment:
If this is only used in the web UI, I would move it closer to where it is used and avoid polluting the `types` package if we can.